spec/graphql/mutations/account/resend_unlock_account_email_mutation_spec.rb

```diff
@@ -53,7 +52,7 @@
       end
 
       it 'does not send an email' do
-        expect(UsersMailer.deliveries).to be_empty
```

Comment:
Please don't delete this check. I'd still like to know if all emails have actually been delivered.